extensions/wikia/Email/i18n/UserRightsChanged.i18n.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$messages = array();
```

Comment:
You're right, they should be. But the crowdin script creates these and changes them back to `array()` whenever we re-download messages. So for these i18n files we let the rule slide.